lib/index.js

```diff
@@ -192,6 +202,37 @@ KnexMigrator.prototype.migrateTo = function migrateTo(options) {
     });
 };
 
+/**
+ * will delete the target database
+ *
+ * @TODO:
+ * - think about deleting only the tables
```

Comment:
> 1. deleting and recreating the whole database (and connection) between every test will result in slower tests for not much value

I've run the Ghost integration tests with both approaches: deleting all tables vs drop and re-create the database. This is the result:
498 passing (7m)
498 passing (7m)

Both the same speed. And the logic for deleting all the tables is way heavier (**plus we need to run these commands in a transaction**).

```
return connection.raw('show tables')
            .then(function (tables) {
                return Promise.each(tables[0], function (table) {
                    return connection.raw('SET FOREIGN_KEY_CHECKS=0;')
                        .then(function () {
                            return connection.raw('DROP TABLE IF EXISTS ' + table['Tables_in_' + dbConfig.connection.database] + ';');
                        })
                        .then(function (resp) {
                            return connection.raw('SET FOREIGN_KEY_CHECKS=1;');
                        });
                });
            })
```

So as you suggested, offering a reset hook is a good alternative for projects wanting to control deleting tables from the database. I would suggest: we keep dropping the database and i will add the reset hook before we go live. What do you think?
If you still want to drop the tables, i am fine with this.